__tests__/getAllUsers.test.js

```diff
@@ -26,7 +26,19 @@ describe('Testing GET users endpoint', () => {
         done();
       });
   });
-
+  it('should get all articles successfully', (done) => {
+    chai
+      .request(app)
+      .get(url)
+      .set('Authorization', `Bearer ${validToken}`)
+      .end((err, res) => {
+        expect(res.status).to.eql(200);
+        expect(res.body.users).to.have.property('message');
+        expect(res.body.users.message).to.eql('Users fetched successfully');
+        expect(res.body.users).to.be.a('object');
```

Comment:
Don't we have at least one specific user you are returning?

Reply:
I don't think there's a way I can ascertain the specific user. can you please add more color @mazma1

Reply:
Your response message says `Users fetched successfully`. So which users did you fetch?

Reply:
Oh Okay, I would work on the test now.